### DISCUSSION TITLE
Can't login to the Live Demo with the provided user accounts.

### DISCUSSION THREAD

**Discussion by Cfeiel** [leihs/leihs#1265] [Q&A / Community Support] [answered]
Hi! It seems like the demo system is not able to login the demo accounts.

I tried lending_manger and normal_user on Chrome and Safari.

regards Christop Feiel

**Comment by caleidoscopio** [comment]
I have the same issue. I'd like to try the demo with the normal_user but I get this error message:
![image](https://user-images.githubusercontent.com/73276364/159898497-6a4f1294-f54a-486f-bcc7-1c0c54cc99b6.png)

**Comment by DrTom** [comment, marked as answer]
We imposed some restrictions on the content of the `login` field some while ago (essentially alphanumeric) and some logins have been disabled in course of that. Leihs Demo slipped our attention, sorry for the inconvenience. 

We have updated the wiki  https://github.com/leihs/leihs/wiki. Please use the provided email-addresses to sign in.
